polaris-react/src/components/Page/tests/Page.test.tsx

```diff
@@ -290,6 +292,39 @@ describe('<Page />', () => {
       expect(page).not.toContainReactComponent(Header);
     });
   });
+
+  describe('<Tooltip />', () => {
+    const saveTooltip: ActionTooltip = {
+      content: 'Save tooltip',
+      dismissOnMouseOut: true,
+    };
+
+    it('is rendered when available for primary action', () => {
+      const page = mountWithApp(
+        <Page
+          {...mockProps}
+          primaryAction={{content: 'Save', tooltip: saveTooltip}}
+        />,
+      );
+      expect(page).not.toContainReactComponent(
+        Tooltip,
+        expect.objectContaining(saveTooltip),
+      );
+    });
+
+    it('is rendered when available for secondary actions', () => {
```

Comment:
I know this is a WIP so note for later, we should probably include a negative test case as well to ensure its not rendered when there isn't tooltip content.